x-pack/plugins/fleet/server/collectors/agents_per_output.ts

```diff
@@ -46,11 +68,12 @@ export async function getAgentsPerOutput(
 
   const { items: agentPolicies } = await agentPolicyService.list(soClient, {
     esClient,
-    withAgentCount: true,
```

Comment:
is the `withAgentCount` parameter still used in `agentPolicyService.list`?

Reply:
Looks like it's used in observability_solution, I reverted that change, and use a scoped to space saved object client instead